go.mod

```diff
@@ -207,3 +206,6 @@ require (
 // We want to format raw bytes as hex instead of base64. The forked version
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
+
+// Note this is a temproary replace and will be removd when taprpc is tagged.
+replace github.com/lightninglabs/taproot-assets/taprpc => ./taprpc
```

Comment:
Hmm, I guess a `go.work` file wouldn't really help us here, at least not much. It would allow us to not have this `replace` directive. But any external project depending on this one would still need to pull the git hash/version from here, so we still need to push a tag and update the `go.mod` file regularly...

This makes everything quite cumbersome, especially in `lightning-terminal`... But I don't really have a better solution...